### ISSUE TITLE
Inspect remaining functions in rank_object_enumerator

### ISSUE THREAD

**Issue by tlamonthezie** [DARMA-tasking/LB-analysis-framework#401]
Check if we want to 
- keep functions used only by the rank_object_enumerator module
- remove the rank_object_enumerator module ?

**Comment by tlamonthezie** [comment]
> Check if we want to
> 
> * keep functions used only by the rank_object_enumerator module
> * remove the rank_object_enumerator module ?

@ppebay with #386 we moved compute_pairwise_reachable_arrangements function to lbsStatistics
Regarding the rank_object_enumerator if we decide to keep it we might close this issue ?

**Comment by ppebay** [comment]
I agree that we can close the issue @tlamonthezie; please verify it's not breaking anything and close issue (as you are the issue creator)

**Comment by tlamonthezie** [comment]
@ppebay the `rank_object_enumerator` script has been removed by the following PR https://github.com/DARMA-tasking/LB-analysis-framework/pull/408/files
The functions that were called by the `rank_object_enumerator` script seems to be npt called anymore
Statistics::compute_all_reachable_arrangements
Statistics::recursively_compute_transitions
Should we remove it ?

**Comment by ppebay** [comment]
This code is no longer used
